### PR TITLE
Use Logger in extension.ts and Utils dir

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This commit changes `console.log` in extension.ts and Utils directory
to `Logger`.

for #662

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

File: src/Utils/Helpers.ts [hunk 1]
```diff
@@ -18,10 +18,14 @@ import * as fs from 'fs';
 import * as path from 'path';
 import * as vscode from 'vscode';
 
+import {Logger} from '../Utils/Logger';
```
Comment (seanshpark) [inline src/Utils/Helpers.ts:21]: maybe just `./Logger` ?

Comment (hyunsik-yoon) [reply src/Utils/Helpers.ts]: Thanks!

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by yunjayh — APPROVED** [review]
LGTM

**Pull request merged**